src/User/pages/Cancellation/Cancellation.jsx

```diff
@@ -34,8 +34,8 @@ const CancellationPage = () => {
             <option value="no_longer_needed">Item no longer needed</option>
             <option value="poor_service">Poor customer service</option>
             <option value="high_shipping">High shipping costs</option>
-            <option value="product_quality">Concerns about product quality</option>
-            <option value="change_circumstances">Change in personal circumstances</option>
+            <option value="product_quality">REceived Damaged Product</option>
```

Comment:
Fix "Received" speeling

Reply:
@codervivek5 Updated the requested changes. Kindly verify. Thank you.